### ISSUE TITLE
`unused-void-attr` complains for no reason

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#637]
This is my code:

```
[line widths] > tabular
  joined. > @
    QQ.txt.text " "
    mappedi.
      QQ.structs.list
        split.
          QQ.txt.text line
          ","
      [p i] >>
        fixed > @
          p
          widths.at i
```

I get this:

```
The void attribute "widths" is unused anywhere in the object "tabular" (unused-void-attr (Single))
```

However, this attribute is used. Looks like false positive warning.

**Comment by bibonix** [comment]
I am planning to work on this issue. I will trace through the `unused-void-attr` lint to understand why `widths` references inside the inner `[p i]` formation are not being recognized as usages.

**Comment by bibonix** [comment]
PR #855 submitted with the fix and a regression test (`allows-void-attr-used-in-nested-formation.yaml`).

The lint now matches `ξ(.ρ)*.<attr>` references, counts the `ρ` hops, and verifies that the formation declaring the void attribute is exactly that many enclosing abstract scopes above the usage. The previous `catches-usage-as-nested-rho-ref` pack was locking in the bug and is renamed/flipped to `allows-usage-as-nested-rho-ref` since `^.x` from a nested formation is now correctly recognized as a usage of the outer `x`.

**Comment by VasilevNStas** [comment]
@yegor256 @volodya-lombrozo @maxonfjvipon, this false-positive is **already resolved on `master`**: the `unused-void-attr` lint has been removed in commit `54fa0bd4` (refactor(#574)). No `unused-void-attr` XSL file exists in `src/main/resources` anymore, so the warning you hit (`"widths" is unused`) can no longer be produced.

I'd suggest closing this as resolved-by-removal.

**Comment by volodya-lombrozo** [comment]
@yegor256 We don't have `unused-void-attr` anymore. Closing this one. Thank you for reporting!
